app/views/alaveteli_pro/info_requests/_embargo_extension_form.html.erb

```diff
@@ -1,7 +1,8 @@
 <%= form_for(
-      [AlaveteliPro::EmbargoExtension.new(embargo: info_request.embargo)],
+      :alaveteli_pro_embargo_extension,
+      url: alaveteli_pro_embargo_extensions_path,
       html: {class: 'js-embargo-form'}) do |f| %>
-  <%= f.hidden_field :embargo_id %>
+  <%= f.hidden_field :embargo_id, value: info_request.embargo&.id %>
```

Comment:
I think we can rely on an embargo ID existing here, since [we check that there is an embargo](https://github.com/mysociety/alaveteli/blob/develop/app/views/alaveteli_pro/info_requests/_embargo_form.html.erb#L13-L15) before rendering this partial.

The controller action [requires an `embargo_id` param](https://github.com/mysociety/alaveteli/blob/develop/app/controllers/alaveteli_pro/embargo_extensions_controller.rb#L10-L11) too, so I think this adds a little confusion.